zerver/migrations/0121_text_choice_in_emojiset.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+# change emojiset to text if emoji_alt_code is true.
+def change_emojiset(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    users = UserProfile.objects.all()
+    for user in users:
+        if user.emoji_alt_code:
+            user.emojiset = "text"
+            user.save(update_fields=["emojiset"])
```

Comment:
I think you can do this better with `for user in UserProfile.objects.filter(emoji_alt_code=True)`.